tests/verify_vector_table.py

```
@@ -0,0 +1,134 @@
+    parser.add_argument(
+        "binary",
+        type=Path,
+        help=(
+            "Path to the AVR ELF or object file that contains the interrupt vector table."
+        ),
+    )
+    return parser.parse_args()
+
+
+def load_vectors_section(binary_path: Path) -> list[str]:
+    result = subprocess.run(
```

Ruff (0.14.0)

`subprocess` call: check for execution of untrusted input

(S603)